x-pack/performance/journeys/cloud_security_dashboard.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { Journey } from '@kbn/journeys';
+import expect from '@kbn/expect';
+
+export const journey = new Journey({
+  beforeSteps: async ({ kibanaServer, retry }) => {
+    retry.try(async () => {
```

Comment:
```suggestion
    await retry.try(async () => {
```

I guess it might be the reason of flakiness.